flink-tests/src/test/java/org/apache/flink/test/recovery/AbstractTaskManagerProcessFailureRecoveryTest.java

```diff
@@ -173,10 +140,6 @@ public void testTaskManagerProcessFailure() throws Exception {
 			taskManagerProcess2 = new ProcessBuilder(command).start();
 			new CommonTestUtils.PipeForwarder(taskManagerProcess2.getErrorStream(), processOutput2);
 
-			// we wait for the JobManager to have the two TaskManagers available
-			// since some of the CI environments are very hostile, we need to give this a lot of time (2 minutes)
-			waitUntilNumTaskManagersAreRegistered(jmActor, 2, 120000);
```

Comment:
We can skip this waiting part now because it is a standalone session cluster? Just double checking.

Reply:
Yes, we can skip this part, because we support now queued scheduling where the TMs don't need to be present before you start the job.